src/test/test_nvmeof_mon_encoding.cc

```diff
@@ -50,9 +50,9 @@ void test_NVMeofGwMap() {
   pending_map.created_gws[group_key]["GW2"].nonce_map[2] = new_nonces;
   dout(0) << " == Dump map before Encode : == " <<dendl;
   dout(0) << pending_map << dendl;
-
+  uint64_t features = 4540701547738038271;
```

Comment:
maybe using an hex number to initialize? would save the reader the need to fire up a calculator...